internal/report.go

```diff
@@ -434,6 +458,38 @@ func IgnoredStatements(conv *Conv) (l []string) {
 	return l
 }
 
+func reportNameChanges(conv *Conv, w *bufio.Writer) {
+
+	w.WriteString("-----------------------------------------------------------------------------------------------------\n")
+	w.WriteString("Name Changes in Migration\n")
+	w.WriteString("-----------------------------------------------------------------------------------------------------\n")
+	fmt.Fprintf(w, "%25s %15s %25s %25s\n", "Source Table", "Change", "Old Name", "New Name")
+	w.WriteString("-----------------------------------------------------------------------------------------------------\n")
+
+	for srcTableName, spTable := range conv.ToSpanner {
+		if srcTableName != spTable.Name {
+			fmt.Fprintf(w, "%25s %15s %25s %25s\n", srcTableName, "Table Name", srcTableName, spTable.Name)
+		}
+		for srcColName, spColName := range spTable.Cols {
+			if srcColName != spColName {
+				fmt.Fprintf(w, "%25s %15s %25s %25s\n", srcTableName, "Column Name", srcColName, spColName)
+			}
+		}
+		for srcFkName, spFkName := range conv.ToSpannerFkIdx[srcTableName].ForeignKey {
```

Comment:
So as part of the bug, we only plan to cover the implicit changes in this. HarbourBridge does not do these ordering changes via the CLI so we dont need to cover that yet. We can take it up during part 2 of this bug when we make the changes for UI logging.

Reply:
![Screenshot 2022-06-13 12 36 55 PM](https://user-images.githubusercontent.com/53526877/173298597-881c531c-6167-4f26-9d42-b3d9a092598e.png)
![Screenshot 2022-06-13 12 35 21 PM](https://user-images.githubusercontent.com/53526877/173298624-244ee559-599c-4440-b1d3-d42e26fb5bb1.png)

In case the foreign key and indexes cannot be mapped to Spanner from the source DB, their report is already present in the codebase.